lib/CL/devices/devices.c

```diff
@@ -114,6 +142,43 @@ static unsigned devices_active = 0;
 
 static pocl_lock_t pocl_init_lock = POCL_LOCK_INITIALIZER;
 
+static void * pocl_device_handles[POCL_NUM_DEVICE_TYPES];
+
+static void
+getPoclDeviceLibPath (char *result, char *device_name)
+{
+  Dl_info info;
+  if (dladdr ((void *)getPoclDeviceLibPath, &info))
+    {
+      char const *soname = info.dli_fname;
+      strcpy (result, soname);
+      char *last_slash = strrchr (result, '/');
```

Comment:
We do not have a Windows maintainer, but does using / directly instead of \ break there?

Reply:
Fixed